lambda-runtime/src/AWS/Lambda/Runtime.hs

```diff
@@ -0,0 +1,47 @@
+module AWS.Lambda.Runtime
+  ( module Exports
+  , Client.InternalLambdaClientError (..)
+  , run
+  )
+where
+
+
+import AWS.Lambda.Env             as Exports
+import AWS.Prelude
+import Control.Monad.Except (runExceptT)
+
+import qualified AWS.Lambda.Client as Client
+import qualified Data.Aeson        as JSON
+
+run
+  :: forall e env m . (MonadCatch m, MonadIO m, Exception e)
+  => env
+  -> (JSON.Value -> RIO env JSON.Value)
+  -> m ()
+run env lambdaFn = forever . runtimeLoop @e $ (runRIO env . lambdaFn)
+
+runtimeLoop
+  :: forall e m . (MonadIO m, MonadCatch m, Exception e)
+  => (JSON.Value -> m JSON.Value)
+  -> m ()
+runtimeLoop fn =
+  either throwM runFunction =<< liftIO (runExceptT Client.getHttpConfig)
+  where
+    runFunction :: Client.HTTPConfig -> m ()
+    runFunction httpConfig = do
+      Client.LambdaResponse{..} <- processNextLambdaAction httpConfig
+        $ Client.getNextLambdaResponse httpConfig
+
+      let action = Client.sendEventSuccess httpConfig requestId
+                       =<< fn event
```

Comment:
__question:__ Not sure what the alignment rule is here.

Reply:
```
      let action = Client.sendEventSuccess httpConfig requestId
                 =<< fn event
```

The alignment above is preferred and I have checked it out and it works but I remember as if that kind of alignment used to cause a GHC parser error. Maybe I miss-remebered.